en/docs/tutorials/create-rest-api-component.md

```diff
@@ -0,0 +1,299 @@
+
+The REST API you designed is currently available only in the VS Code Editor. To use it, you need to save it in the Choreo Console. You can do this by committing the REST API configuration into a private repository in GitHub as follows:
+
+1. Click **Sync with Choreo Upstream** (at the bottom of the page), and then click **Sync my changes with Choreo**.
+
+2. In the left panel, enter `Implement REST API` as the message and click on the tick.
+
+    ![Commit message](../assets/img/tutorials/rest-api/commit-message.png){.cInlineImage-half} 
+
+    Select **Yes** in the message that appears to specify that you need the changes to be staged.
+
+Once the changes are successfully deployed, the VS Code Editor indicates by displaying the text **In sync with Choreo upstream** for the `service.bal` file.
+
+
+## Deploy
+
+To deploy the API, click the **Deploy** icon. Then in the **Build Area** card, click **Deploy**.
+
+![Deploy API](../assets/img/tutorials/rest-api/deploy-api.png){.cInlineImage-full}
+
+To check the progress of the deployment, click **View Logs**
+
+![Deployment progress](../assets/img/tutorials/rest-api/deployment-progress.png){.cInlineImage-full}
+
+Once the API is deployed, the **Development** card indicates that the API is running as shown below.
+
+![Deployed API](../assets/img/tutorials/rest-api/deployed-api.png){.cInlineImage-full}
+
+Now you can test your deployed REST API to check again whether it is working as expected.
+
+## Test
+
+Once you have deployed the Choreo application you can test it via the Open API Console, A cURL command, or Postman.
+
+For this scenario, let's test via the Open API Console:
+
+1. To open the test view, click **Test** in the **Development** card in the **Deploy tab, or click on the **Test** tab.
+
+2. Click **Try it out**.
+
+3. In the **country** field, enter `USA`.
+
+4. Click **Execute**.
+
+The following is displayed as the response body under **Responses**.
+
+![Response for the Deployed API](../assets/img/tutorials/rest-api/deployed-api-response.png){.cInlineImage-full}
+
+Congratulations! You have now successfully created and tested a REST API in Choreo!
+
+## Manage
+
+Once you have successfully deployed your API and tested it, you can perform the following actions for it
+
+- Manage the lifecycle: You can publish your API to the Developer Portal to make it available for public use, or deploy it as a prototype so that users can try it out and provide feedback for improvement. Once you publish an API, you can deprecate it or block it from being used if required.
+
+- Validate subscriptions: You can configure subscription validation to mandate subscriptions.
+
+- Add documents: This involves attaching files with information about the API for users.
+
+- Select/switch usage plans: You can select a usage plan for your API out of `Bronz` `Silver`, `Gold`, and `Unlimited` based on the level of traffic that you expect the API to receive. You can change the usage plan when required.
```

Comment:
Bronze

Reply:
Since we don't have a tutorial for Managing the REST API, should we add it her for now?

Reply:
This is under discussion. We might in fact remove this section for now.